source/includes/options-mongodrdl.yaml

```diff
@@ -270,5 +270,36 @@ description: |
 
   Specifies the database where the schema information is stored. This
   option is not used by the :commandoption:`sample <sample>` command.
+---
+program: mongodrdl
+name: uri
+directive: option
+args: <mongodb-uri>
+description: |
+  .. versionadded:: 2.12
 
+  MongoDB URI connection string.
+optional: true
```

Comment:
The new `--uri` flag in `mongodrdl` cannot be specified in conjunction with some other flags. Specifically, it follows the `--uri` flag rules listed here for mongo tools: https://docs.mongodb.com/manual/reference/program/mongorestore/#cmdoption-mongorestore-uri (see the "Important" warning box).

Is that information you should include in this PR?

Reply:
Yes indeed, thanks.